7_react/react1/week1/ToDoList/src/Components/TaskInput.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import "../index.css";
+
+function TaskInput() {
+  const [toDoList, setToDoList] = useState([]);
+  const [todo, setTodo] = useState("");
+
+  console.log(todo);
```

Comment:
console logs used for solely debugging purposes should not be pushed to the repository, please remove :)